CONTRIBUTING.md

```diff
@@ -0,0 +1,90 @@
+# Contributing to voila
+
+## Setting up a development environment
+
+```bash
+# create a new conda environment
+conda create -n voila -c conda-forge notebook nodejs
+conda activate voila
+
+# install voila in editable mode
+python -m pip install -e .
+```
+
+## Run voila
+
+To start voila, run:
+
+```bash
+voila
+```
+
+or
+
+```bash
+python -m voila
+```
+
+This will open a new browser tab at [http://localhost:8866/](http://localhost:8866/).
+
+When making changes to the frontend side of voila, open a new terminal window and run:
+
+```bash
+cd js/
+npm run watch
+```
+
+Then reload the browser tab.
+
+## Extensions
+
+To manually enable the classic notebook server extension:
+
+```bash
+jupyter serverextension enable voila --sys-prefix [--py | --user]
+```
+
+For Jupyter Server:
+
+```bash
+jupyter extension enable voila --sys-prefix [--py | --user]
```

Comment:
remove `--user`.

Reply:
It seems to me that `--py` is always required?

Reply:
`--py` throws the following for me:

```
KeyError: 'The Python module voila does not include any valid server extensions'
```

Reply:
Full command: `jupyter serverextension enable voila --sys-prefix --py`

Reply:
indeed we don't have an entry point for a jupyter server extension in `__init__.py`. We should probably just remove the `--py`.